libtransmission/rpc-server.c

```diff
@@ -560,12 +560,31 @@ static void handle_request(struct evhttp_request* req, void* arg)
 
     if (req != NULL && req->evcon != NULL)
     {
+        static int attempts = 0;
         char const* auth;
         char* user = NULL;
         char* pass = NULL;
 
         evhttp_add_header(req->output_headers, "Server", MY_REALM);
 
+        if (attempts == 100)
+        {
+            send_simple_response (req, 403,
```

Comment:
This is minor and I could fix it up myself but note that space between function name and the brace isn't the style anymore. Same below with a couple of `tr_free` calls.